src/app.module.ts

```diff
@@ -16,6 +17,7 @@ import { AppService } from './app.service';
         username: configService.get('POSTGRES_USER'),
         password: configService.get('POSTGRES_PASSWORD'),
         database: configService.get('POSTGRES_DB'),
+        entities: [User],
```

Comment:
You dont need to load entities here since you have `autoLoadEntities: true`.

Reply:
Resolved on 3aa3aae6f20eeaff72b35bbe93b213801d9ce268.